### PR TITLE
Metrics for replacements DD

### PR DESCRIPTION
**Pull request by xurui-c**
As part of the roll out plan of automatically skipping offending replacements projects, we want to create a DD dashboard to monitor the following metrics:
- number of projects skipped
- latency of adding skipped projects to redis
- latency of deleting expired projects from redis
- latency of getting skipped projects from redis

### CODE REVIEW

File: snuba/replacers/replacements_and_expiry.py [hunk 1]
```diff
@@ -8,8 +8,12 @@
 logger = logging.getLogger(__name__)
 
 
+from snuba import environment
 from snuba.redis import RedisClientKey, get_redis_client
 from snuba.state import get_int_config
+from snuba.utils.metrics.wrapper import MetricsWrapper
+
+metrics = MetricsWrapper(environment.metrics, "bucket_timer")
```
Comment (volokluev) [inline snuba/replacers/replacements_and_expiry.py:16]: Why is it called `bucket_timer`?

File: snuba/replacers/replacements_and_expiry.py [hunk 2]
```diff
@@ -24,6 +28,7 @@ def set_config_auto_replacements_bypass_projects(
 ) -> None:
     try:
         projects_within_expiry = get_config_auto_replacements_bypass_projects(curr_time)
+        start = datetime.now()
```
Comment (volokluev) [inline snuba/replacers/replacements_and_expiry.py:31]: use `time.time()` instead

Comment (volokluev) [reply snuba/replacers/replacements_and_expiry.py:31]: that will give you the measurement in seconds already

File: snuba/replacers/replacements_and_expiry.py (hunk 1 above)
Comment (xurui-c) [reply snuba/replacers/replacements_and_expiry.py]: Corrected to `replacements_and_expiry`

**Review by volokluev — APPROVED** [review]
also think it would be good to output a metric whenever anything fails

**Pull request merged**